components/About/styles.module.scss

```diff
@@ -25,6 +43,7 @@
     background-color: #f7dea6;
     width: 344px;
```

Comment:
(also applies to the text elements)
When dealing with mobile responsiveness it's good to avoid fixed width values. In this case you could do something like: `max-width: 344px` instead so that the image won't go off screen on tiny phones (just check down to 320px width)